server/app/assets/javascripts/admin_import_view.ts

```diff
@@ -3,13 +3,30 @@ import {attachRedirectToPageListeners} from './main'
 class AdminImportView {
   constructor() {
     this.addRedirectListeners()
+    this.addStringifyJsonListener()
   }
 
   addRedirectListeners() {
     document.addEventListener('htmx:afterSwap', () => {
       attachRedirectToPageListeners()
     })
   }
+
+  addStringifyJsonListener() {
```

Comment:
I couldn't think of a way to test the changes I'm making here. Lmk if you have ideas!